publisher-sdk/src/main/java/com/criteo/publisher/adview/AdWebViewClient.kt

```diff
@@ -61,7 +63,8 @@ class AdWebViewClient(
 
         WebResourceResponse("text/javascript", "UTF-8", stream)
       } catch (e: IOException) {
-        throw RuntimeException(e)
+        logger.log(MraidLogMessage.onErrorDuringMraidFileInject(e))
```

Comment:
The last commits is overriding some things that were done in the "Inject mraid scrpt" commit like if this previous commit was simply a WIP.

Maybe it is better to squash them together ? You need the test to fixup the implementation, so both test and implementations are coming together naturally.

Also, on a general note, do you mind doing the transformation from Java to Kotlin in a dedicated commit (ie: one commit for the renaming to keep git history + 1 commit for the actual migration + 1 commit for your added feature) ? So it will ease the review and the MRAID commit will be much smaller.

Reply:
Sure, will cleanup my commits!

Reply:
Updated